docs/contracts/node-operators-registry.md

```diff
@@ -195,6 +617,39 @@ Returns the operator struct based on the operator owner address
 | --------------  | --------- | ----------------------------------                                  |
 | `_user`   | `address` | Address of the operator owner|
 
+
+### getValidatorShare()
```

Comment:
Make sure that we have not duplicate documentation, this function was added 2 times